docs/template_plugin/backend/evaluates_map.cpp

```diff
@@ -3589,7 +3589,7 @@ template <element::Type_t ET>
 bool evaluate(const shared_ptr<op::v9::GenerateProposals>& op,
```

Comment:
maybe it's not a question of this PR, but can we finally split this file into pieces? Currently, it takes too much time to compile

Reply:
I agree - it's huge. I see two ways of such split:
1. Split by chosen category (opset or operator). That implies splitting `docs/template_plugin/backend/opset_int_tbl.hpp`.
2. Create template declarations and move definitions into separate files. That implies template instantiations for each type and op.